qemu-test-init

```diff
@@ -218,13 +221,11 @@ if [ -n "$SHELL" ]; then
   setsid bash -c "$BASH_CMD" </dev/ttyS0 >/dev/ttyS0 2>&1 || echo exit-code=$?
   save_gcov_data
   echo o > /proc/sysrq-trigger
+  sleep 1
 fi
 
-if [ -n "$SERVICE" ]; then
-  rauc status mark-good
-fi
-
-if meson test -C $BUILD_DIR "$MESON_TEST" -v; then
+eval "set -- $test"
+if meson test -C $BUILD_DIR "$@" -v; then
```

Comment:
Wouldn't
```suggestion
if meson test -C $BUILD_DIR $test -v; then
```
behave the same?

Reply:
Indeed.

As long as the variable is not be double-quoted (otherwise the meson command has an empty string parameter and would fail if no test is given).

The double-quoting prevents from globbing. However the globbing happens by the shell on the host (unless escaped).

And my solution does not prevent globbing, it does shut the shellcheck warning without adding a comment.